src/scripts/controller/apiServices.ts

```diff
@@ -0,0 +1,23 @@
+import { Product, ProductResponse } from '../model/interfaces';
```

Comment:
While importing a types we can use `type` keyword which ensures that TypeScript treats the import as a type-only import, preventing unnecessary runtime imports.

```suggestion
import type { Product, ProductResponse } from '../model/interfaces';
```

Reply:
Added `type` keyword (https://github.com/krushna-simform/product-management-system/pull/3/commits/0a20f43723af3fded7c85c3b389bf717aab9364c)